src/AMP.php

```diff
@@ -49,7 +49,9 @@ class AMP
     // The StandardFixPass should be after StandardScanPass
     public $passes = [
         'Lullabot\AMP\Pass\PreliminaryPass', // Removes user blacklisted tags
-        'Lullabot\AMP\Pass\SugarImgTagTransformPass',
+        'Lullabot\AMP\Pass\ImgTagTransformPass',
```

Comment:
I have concerns. Is there a way to not fork library, but extend it? In that case would be enough just to extend this class and override passes, or adjust passes after object creation (field is public).
Also all this pass files could just be inherited from original versions. 
I am generally against forking public library, because it'll be hell to maintain. 
Is there a strong reasons to fork things?